recipes/nanocube/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "nanocube" %}
+{% set version = "0.2.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/nanocube-{{ version }}.tar.gz
+  sha256: 5737404f67c8cc1cc05c308a9fc08d04bf0e99568009dd07b1b3c904e8368863
+  patches:
+    - 0001-get-version.patch
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+  script_env:
+    - VERSION={{ version }}
+
+requirements:
+  host:
+    - python >=3.10
+    - setuptools >=61.0
+    - pip
+  run:
+    - python >=3.10
+    # Pip check complains about numpy version
```

Comment:
For this package or a dependency? If the former, you should report upstream, if the latter, you should report in the feedstock of the dependency. Adding this workaround here is OK but will hide the real problem.

Reply:
Got it, let me investigate

Reply:
I check both `pyproject.toml` and `setup.py` from upstream and it seems both didn't specify a numpy version? Not sure if this is something else upstream or if it's a pip check quirk? @ocefpaf

Reply:
Can you remove that in so we can check the logs?

Reply:
@ocefpaf Yeah I was planning to but looks like we're having some CI issues; waiting for that to resolve.

Reply:
Yep. Keep an eye in https://github.com/conda-forge/status/issues/188

Reply:
They should be back online now.